modules/aws-limits.adoc

```diff
@@ -50,7 +50,7 @@ To use the `us-east-1` region, you must increase the EIP limit for your account.
 |Elastic Load Balancing (ELB/NLB)
 |3
 |20 per region
-|By default, each cluster creates internal and external network load balancers for the primary API server and a single classic elastic load balancer for the router. Deploying more Kubernetes LoadBalancer Service objects will create additional link:https://aws.amazon.com/elasticloadbalancing/[load balancers].
+|By default, each cluster creates internal and external network load balancers for the primary API server and a single classic ELB load balancer for the router. Deploying more Kubernetes LoadBalancer Service objects will create additional link:https://aws.amazon.com/elasticloadbalancing/[load balancers].
```

Comment:
@dfitzmau The following comments are derived from AWS style guidance. 

On first usage of an AWS service name within a topic, AWS style recommends writing out the full name to introduce the acronym (e.g "Elastic Load Balancing (ELB)"). The acronym may be used on subsequent usage, once the full service name is introduced. 

When referring to specific load balancer types, AWS uses the following terminology. These words are always used together in the phrasing seen below, and written in all caps. 

- Classic Load Balancer 
- Application Load Balancer 
- Network Load Balancer 

See: 

https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/introduction.html
https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html
https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html

Reply:
When referring to a generic load balancer and type does not matter (i.e. Classic, Application, or Network), the term "load balancer" is used.

Reply:
I recommend revising as follows: 

"By default, each cluster creates internal and external ELB Network Load Balancers for the primary API server and a single Classic Load Balancer for the router."

Reply:
Thanks, @codymant . I updated the sentence based on your feedback. I updated the component name, `ELB/NLB`, to `Elastic Load Balancing (ELB)`. I removed `NLB` as it seemed to confuse things. WDYT?